src/index.html

```diff
@@ -1,14 +1,58 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#"><img src="./images/logo.png" alt="logo"></a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="list_item">
+            <a class= "is-active Apple" href="#">APPLE</a>
+          </li>
+
+          <li class="list_item">
+            <a href="#">SAMSUNG</a>
+          </li>
```

Comment:
1. name classes using BEM methodology
2. use lowercased text values at html and change this case at css
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a class= "nav__link is-active" href="#">apple</a>
          </li>

          <li class="nav__item">
            <a href="#" class="nav__link">samsung</a>
          </li>
```

Reply:
тоді як це застосувати для L&C коли вибиває помилку: "Only 2 attributes per line are permitted"